pkgs/by-name/fr/frog-protocols/package.nix

```
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  lib,
+  meson,
+  ninja,
+  nix-update-script,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "frog-protocols";
+  version = "0.01-unstable-2024-09-25";
+
+  src = fetchFromGitHub {
+    owner = "misyltoad";
+    repo = "frog-protocols";
+    rev = "17be81da707722b4f907c5287def442351b219b0";
+    hash = "sha256-N8a+o5I7CRoONCvjMHVmPkJTVncczuFVRHEtMFzMzss=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+  ];
+
+  passthru.updateScripts = nix-update-script { };
```

```suggestion
  passthru = {
    tests.pkg-config = testers.hasPkgConfigModules { package = finalAttrs.finalPackage; };
    updateScripts = unstableGitUpdater { };
  };
```

This can also be a quick smoke test. It'll ensure the version matches in stable releases as well